lib/runtime-core/src/instance.rs

```diff
@@ -58,6 +61,11 @@ pub struct Instance {
 }
 
 impl Instance {
+    /// Access the new exports API.
+    /// TODO: rename etc.
+    pub fn exports_new(&self) -> Exports {
```

Comment:
Can we rename it to `exports_map` for now?

Reply:
Sure!  I have no intention of shipping with the name `exports_name`, it's just a placeholder for now